### ISSUE TITLE
Add Travis CI configuration

### ISSUE THREAD

**Issue by jimmysoda** [PDAL/CAPI#1]
Create a Travis CI configuration file (.travis.yml) for building the pdal-c master branch ~~and also creating releases at tagged commits~~.

Refer to the existing GitLab CI configuration file (.gitlab-ci.yml) for an example. If possible, use the Docker images at [simverge/pdalc-build-deps](https://cloud.docker.com/u/simverge/repository/docker/simverge/pdalc-build-deps) for the builds, since they simply extend [pdal/pdal](https://hub.docker.com/r/pdal/pdal) Docker images. Dockerfiles for Alpine and Ubuntu images are in the pdal-c repository's tests/docker directory.

Update the root README.md to report the status of the pdal-c build on a badge.

**Comment by jimmysoda** [comment]
I removed requirement for creating releases at tagged commits. I decided not to distribute binaries via GitHub at the moment. I'll look into vcpkg to distribute pdal-c for Windows. I think I can set up GitHub-based releases without having to configure them in Travis CI.

**Comment by hobu** [comment]
A conda-forge package might also be another convenient way to distribute a binary. The official PDAL binaries are now on conda to give us good dependency coverage and easy user on-boarding.

**Comment by jimmysoda** [comment]
I addressed the issue in the Simverge/pdal-c repository and just pushed master to this repository. I could not get Travis CI to display PDAL/CAPI as an available repository, so I used Simverge/pdal-c instead.

Also, GitHub did not let me create a pull request from Simverge/pdal-c to PDAL/CAPI because the repositories do not have a *fork* relationship. Nevertheless, I pushed the change that addresses this issue to PDAL/CAPI master.
